spec/definitions/DashboardRequest.yaml

```diff
@@ -0,0 +1,27 @@
+type: object
+properties:
+  metrics:
+    type: object
+    properties:
+      metricName:
+        type: object
+        description: Custom metric name which will be used in the response containing the appropriate data.
+        properties:
+          type:
+            type: string
+            description: Type of data and grid of the metric.
+            enum:
+              - approvedThroughput
+              - salesAllCount
```

Comment:
seems like `salesCount` may be better?

Reply:
Transaction report already has `salesCount` but it is about approved sales, and dashboard is about all sales. I wanted this to be more consistent so I didn't find anything better than this.